zingo-testutils/src/lightclient/with_assertions.rs

```diff
@@ -29,6 +29,7 @@ where
 
     environment.bump_chain().await;
 
+    assert_send_outputs_match_sender(client, &proposal, &txids).await;
```

Comment:
I am having a hard time parsing this name.

To me it seems like this is in a mod called `assertions`.. so if I know which mod it lives in, then I know it's an `assertion`.

It seems like this code shows that the LRZ calculated:

`step.balance().fee_required()`

matches the:

`records.calculate_transaction_fee(record)`

Then if there's agreement, that `fee` is returned.

So, this is a fn in a mod name assertions but it does not return a bool.   That's my intuition for a fn in an assertions mod.

I'll propose-as-code.